cmd/mesh/main.go

```diff
@@ -66,7 +66,7 @@ func main() {
 	go func() {
 		defer wg.Done()
 		log.WithField("ws_rpc_addr", config.WSRPCAddr).Info("starting WS RPC server")
-		rpcServer := instantiateServer(ctx, app, config.WSRPCAddr)
+		rpcServer := instantiateServer(ctx, app, config.WSRPCAddr, "WS")
```

Comment:
I don't like passing around magic strings like this. As a reader and potential caller of `instantiateServer`, it is difficult to understand what the string does and which values are allowed/accepted. Can we either:

1. Infer the type of server from the given address.
2. Create two separate functions which share common functionality. (e.g. `instantiateHTTPServer` and `instantiateWSServer`).

Reply:
Good point. I will keep this in mind in the future.